docs: translate blogs on 10/2021 (#679)
iresty/iresty.github.io: website/blog/2021/10/14/weekly-report-1014.md

---
title: Community Weekly｜10.1-10.14 Feature Highlight Updates in Progress
keywords:
- Apache APISIX
- Community Weekly
- APISIX
- API Gateway
- Contributor
description: The Apache APISIX Community Weekly Newsletter hopes to help community members better understand the weekly progress of the Apache APISIX community and facilitate your participation in the Apache APISIX community.
tags: [Events]
---
> From 10.1 to 10.14, 27 developers have committed 67 commits to Apache APISIX. Thank you to these folks for making the Apache APISIX project better with your selfless efforts!

> <! --truncate-->

Reviewer: ```suggestion
<!--truncate-->
```

## Introduction

Apache APISIX has grown as a community since the first day of open source and has quickly become the most active open source API gateway project in the world. These achievements could not have been achieved without the joint efforts of our community partners.

> "The Apache APISIX Community Weekly Newsletter hopes to help community members better grasp the weekly progress of the Apache APISIX community and facilitate your participation in the Apache APISIX community.

Reviewer: ```suggestion
"f you want to go fast, go alone.If you want to go far, go together", Apache APISIX Community Weekly Report
 hopes to help community members better grasp the weekly progress of the Apache APISIX community and facilitate your participation in the Apache APISIX community.
```

We've also put together some issues for those new to the community!

## Contributor statistics

! [This week's contributor list](https://static.apiseven.com/202108/1634183939241-a87516e5-cb52-4532-87e2-306c09155a70.png)

! [New contributors this week](https://static.apiseven.com/202108/1634183019951-bcf250cd-e5b5-443e-afc4-3cfdef0d6eab.jpg)

## Good first issue

### Issue #5165

**Link**: https://github.com/apache/apisix/issues/5165

> **problem description**.

Reviewer: ```suggestion
**Problem Description**:
```
